### PR TITLE
Hatch 301/filter enums

### PR DESCRIPTION
**Pull request by CMLabay**
hatch-301

### CODE REVIEW

File: packages/rest-client/src/services/utils/schema/schema.ts [hunk 1]
```diff
@@ -67,6 +71,8 @@ export function transformSchema(schema: OldSchema): Schema {
     resolved.attributes[key] = {
       type: transformDataType(stringValue),
       allowNull,
+      ...(typeof value !== "string" &&
+        "values" in value && { values: value.values }),
```
Comment (arthurpankiewicz) [inline packages/rest-client/src/services/utils/schema/schema.ts:75]: can you explain whats going on here? 😅

File: packages/design-mui/src/components/MuiList/components/MuiBody.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { Checkbox, TableBody, TableCell, TableRow } from "@mui/material"
 import type { HatchifyDisplay, XCollectionProps } from "@hatchifyjs/react-ui"
-import MuiBodySkeleton from "./MuiBodySkeleton"
+import { MuiBodySkeleton } from "./MuiBodySkeleton"
```
Comment (arthurpankiewicz) [inline packages/design-mui/src/components/MuiList/components/MuiBody.tsx:3]: this should be a default import i think

Comment (CMLabay) [reply packages/design-mui/src/components/MuiList/components/MuiBody.tsx:3]: yeah, you're right. There was a warning that I misunderstood

**Review by arthurpankiewicz — APPROVED** [review]
👍

**Pull request merged**